sycl/test-e2e/KernelFusion/abort_internalization_stored_ptr.cpp

```diff
@@ -1,11 +1,12 @@
 // REQUIRES: fusion
 // RUN: %{build} -fsycl-embed-ir -o %t.out
-// RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not "Computation error" --implicit-check-not "Internalized"
+// RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not "Computation error" --implicit-check-not "Internalized" --check-prefix=CHECK %if ext_oneapi_hip %{ --check-prefix=CHECK-HIP %} %else %{ --check-prefix=CHECK-NON-HIP %}
 
 // Test pointers being stored are not internalized.
 
 // CHECK:      Unable to perform all promotions for function {{.*}}. Detailed information:
-// CHECK-NEXT: Failed to promote argument 0 of function {{.*}}: It is not safe to promote values being stored to another pointer
+// CHECK-NON-HIP-NEXT: Failed to promote argument 0 of function {{.*}}: It is not safe to promote values being stored to another pointer
+// CHECK-HIP-NEXT: Failed to promote argument 0 of function {{.*}}: Do not know how to handle value to promote
```

Comment:
Is this a temporary restriction?

Reply:
No, from what I saw this is due to the code injected by libclc, so not really something I can easily control.

Reply:
What is the overall status of internalization for the HIP target?

Reply:
same as CUDA, here it bails out because it hits another instruction before the one hit on other targets